README.md

```diff
@@ -457,6 +460,29 @@ export default antfu({
 })
 ```
 
+To control which rules apply to your Svelte files, you can apply overrides like this:
+ 
+```js
+// eslint.config.js
+import { antfu } from '@antfu/eslint-config'
+
+export default antfu({
+  svelte: true,
+}).overrides({
+  'antfu/svelte/rules': {
+    rules: {
+      'svelte/no-dom-manipulating': 'error',
+    },
+  },
+  'antfu/typescript/rules': {
+    files: ['**/*.svelte'],
```

Comment:
This would break support for other ts files. I guess it's better to append a separate config that overrides that rule for only `**/*.svelte` but not other ts files.

Reply:
Sorry to be slow. Why would that break support? It only adds `**/*.svelte` to the already existing files array.